oiosaml/src/test/java/dk/itst/oiosaml/sp/model/OIOLogoutRequestTest.java

```diff
@@ -74,29 +70,37 @@ public void testValidateLogoutRequest() throws Exception {
 		}
 		
 		try {
-			lh.validateRequest(URLDecoder.decode(Utils.getParameter("Signature", url), "UTF-8"), url.substring(url.indexOf('?') + 1), credential.getPublicKey(), "dest", "issuer");
+			String parm = Utils.getParameter("Signature", url);
+			assert parm != null;
+			lh.validateRequest(URLDecoder.decode(parm, "UTF-8"), url.substring(url.indexOf('?') + 1), credential.getPublicKey(), "dest", "issuer");
 			fail();
 		} catch (LogoutRequestValidationException e) {
 			assertEquals(2, e.getErrors().size());
 		}
 		
 		try {
-			lh.validateRequest(URLDecoder.decode(Utils.getParameter("Signature", url), "UTF-8"), url.substring(url.indexOf('?') + 1), credential.getPublicKey(),location, "issuer");
+			String parm = Utils.getParameter("Signature", url);
+			assert parm != null;
+			lh.validateRequest(URLDecoder.decode(parm, "UTF-8"), url.substring(url.indexOf('?') + 1), credential.getPublicKey(),location, "issuer");
 			fail();
 		} catch (LogoutRequestValidationException e) {
 			assertEquals(1, e.getErrors().size());
 		}
-		
-		lr.setNotOnOrAfter(new DateTime().minusMinutes(1));
+
+		lr.setNotOnOrAfter(new DateTime().minusHours(1));
```

Comment:
Check with profile, an hour is too much.

Reply:
Yes, default er 5 minutter, så det hjalp at rette det til 5
```
/**
	 * Set this value to modify the clockskew (default is 5 minutes)
	 */
	static final String PROP_CLOCK_SKEW = "oiosaml-sp.clock.skew";
```